ui/raidboss/data/03-hw/trial/sophia-ex.txt

```diff
@@ -1,157 +1,147 @@
-12	"Gnosis" #knockback
-24	"Shadows Spawn" sync /Added new combatant Aion Teleos/ window 35
-30	"Add Tethers" sync /The aion teleos readies Duplicate/ window 30
-29	"Thunder"
-35	"Aero III"
-38	"Duplicate"
-45	"Execute"
-55	"Cloudy Heavens"
-59	"Adds Spawn" sync /Added new combatant The Second Demiurge/ window 70,0
-70	"Divine Spark" #Look Away
-85	"Gnostic Rant" #Behind dragoon
-104	"Infusion"
-
-################# Phase 2
-200	"Scales of Wisdom" sync /Sophia uses The Scales of Wisdom/ window 200,0
-211	"Scale Tethers" #This set is always balanced
-220	"Quasar (on arena)" #before 10 stack
-223	"Aero III"
-226	"Scale Tethers"
-238	"Tilt"
-
-################# Phase 3; Phase change at 76%
-300	"Thunder II" sync /Sophia uses Thunder II/ window 100,0
-303	"Scale Tethers"
-313	"Onrush" sync /Sophia uses Onrush/ window 313,0 #half arena dash + vulnerability
-314	"Tilt"
-333	"Dischordant Damnation" #before 6 stack partner
-334	"Cintamani" #AoE
-336	"Cintamani" #AoE
-343	"Arms of Wisdom" #Tank Buster
-	
-################# Phase 4; Phase change at 67%
-500	"Shadows Spawn" sync /Added new combatant Aion Teleos/ window 200,0
-504	"Cintamani" #AoE
-506	"Cintamani" #AoE
-508	"Cintamani" #AoE
-510	"Add Tethers"
-513	"Thunder"
-518	"Aero III"
-521	"Duplicate"
-528	"Arms of Wisdom" #Tank Buster
-539	"Execute"
-
-
-################# Phase 5; loops until end
-551	"Cintamani" #AoE
-553	"Cintamani" #AoE
-555	"Cintamani" #AoE
-562	"Arms of Wisdom" #Tank Buster
-564	"Scale Tethers"
-576	"Tilt"
-589	"Thunder III"
-598	"Arms of Wisdom" #Tank Buster
-602	"Scale Tethers"
-612	"Tilt"
-614	"Onrush"
-626	"Cintamani" #AoE
-628	"Cintamani" #AoE
-630	"Cintamani" #AoE
-637	"Arms of Wisdom" #Tank Buster
-643	"Thunder II"
-653	"Dischordant Damnation" #before 6 stack partner
-658	"Gnosis" #knockback
-670	"Arms of Wisdom"
-681	"Scale Tethers"
-685	"Dischordant Damnation" #before 6 stack partner
-692	"Tilt"
-696	"Shadows Spawn"
-700	"Cintamani" #AoE
-702	"Cintamani" #AoE
-704	"Cintamani" #AoE
-710	"Arms of Wisdom" #Tank Buster
-714	"Add Tethers"
-716	"Thunder"
-719	"Duplicate"
-720	"Add Tethers"
-722	"Aero III"
-725	"Duplicate"
-735	"Quasar (on arena)" #before 10 stack
-742	"Arms of Wisdom"
-753	"Dischordant Damnation" #before 6 stack partner
-756	"Execute"
-
-#Repeat loop of phase 5
-768	"Cintamani" #AoE
-770	"Cintamani" #AoE
-772	"Cintamani" #AoE
-779	"Arms of Wisdom" #Tank Buster
-781	"Scale Tethers"
-792	"Tilt"
-806	"Thunder III"
-815	"Arms of Wisdom"
-817	"Scale Tethers"
-829	"Tilt"
-831	"Onrush" #half arena dash + vulnerability
-844	"Cintamani" #AoE
-846	"Cintamani" #AoE
-848	"Cintamani" #AoE
-854	"Arms of Wisdom" #Tank Buster
-860	"Thunder II"
-871	"Dischordant Damnation" #before 6 stack partner
-876	"Gnosis" #knockback, stack SE
-888	"Arms of Wisdom"
-899	"Scale Tethers"
-903	"Dischordant Damnation"
-910	"Tilt"
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start"
```

Comment:
Can you add an Engage! sync as well?

Reply:
Can do. Is there a particular reason I couldn't do a small modification to `make_timeline` so that's automatic? I feel like I've forgotten that in a number of recent timelines. >_>

Reply:
Sounds reasonable!